api/src/services/assets.ts

```diff
@@ -151,7 +151,10 @@ export class AssetsService {
 				const transformer = sharp({
 					limitInputPixels: Math.pow(env.ASSETS_TRANSFORM_IMAGE_MAX_DIMENSION, 2),
 					sequentialRead: true,
-				}).rotate();
+				});
+
+				if(transforms.find((transform) => transform[0] === 'rotate') === undefined)
```

Comment:
I think this line isn't necessary. Just the removal of `rotate()` seems to do the trick as the rotation is applied in the following line. 🤔

Reply:
`rotate` by default aligns the image with it's metadata orientation so we still want to correctly rotate the image when the metadata exists.

Reply:
Got it. Thanks for the clarification! 👍